prost-build/src/lib.rs

```diff
@@ -159,6 +154,10 @@ pub use config::{
     error_message_protoc_not_found, protoc_from_env, protoc_include_from_env, Config,
 };
 
+mod module;
+// HELP(gibbz00): I think `Module` should be made pub(crate).
+pub use module::Module;
```

Comment:
I am not sure. It seems like it is used in this generate call: https://docs.rs/prost-build/latest/prost_build/struct.Config.html#method.generate

I don't understand the use case, but I think we should leave that `generate` method in place.

Reply:
Alright, you're probably right :)